cunumeric/linalg/linalg.py

```diff
@@ -650,6 +652,11 @@ def _solve(
                 f"Output shape mismatch: expected {b.shape}, "
                 f"but found {out.shape}"
             )
+        elif out.dtype != b.dtype:
+            raise TypeError(
+                "Output type mismatch: expected {b.dtype}, "
```

Comment:
Missing f in front of f-string

Reply:
https://github.com/nv-legate/cunumeric/pull/568/commits/e94960b40b8cafa5ff0e1c46926093d1221a489d